tests/e2e/tests/pilot/tcp_test.go

```diff
@@ -74,3 +77,98 @@ func TestTcp(t *testing.T) {
 		}
 	})
 }
+
+func TestTCPNonIstioToIstioHeadlessPort(t *testing.T) {
+	if tc.Kube.AuthEnabled {
```

Comment:
is it expected to skip the test when authN is enabled ?

Reply:
Yes.. coz this is from pod t that has no istio.